release-notes/9.0/supported-os.json

```diff
@@ -294,10 +294,11 @@
                         "x86"
                     ],
                     "supported-versions": [
-                        "23H2",
                         "2022",
                         "2019",
-                        "2016"
+                        "2016",
+                        "2012-R2",
```

Comment:
I'm asking for insight on 2012/R2 on .NET 9 support, from @jamshedd.

Reply:
Same for 2008 then since it's in ESU4: https://learn.microsoft.com/de-de/lifecycle/products/windows-server-2008-r2
Was never listed as supported in the past (also not on the markdown before having the JSON).

Reply:
I got confirmation that we still support 2012/R2, including for .NET 9.

Reply:
Question about 2008 is still open.

Reply:
I was saying it is correct that 2008 was never listed. We never supported it, so it should not be in either supported or unsupported lists. This one isn't a function of ESU. It's that we don't test it and it may not work. We do test Server 2012.

Reply:
Thanks for the clarification. Then this should be fine. EoL.date was merged, so I will have another run of distrossed and the core-support tooling to see if any more might be off. But should be fine to merge otherwise :)